unit-tests/test-fw-update.py

```diff
@@ -72,6 +74,27 @@ def reset_update_counter( device ):
 
     send_hardware_monitor_command( device, cmd )
 
+def find_image_or_exit( product_name, image_mask ):
+    pattern = re.compile('Intel RealSense ')
```

Comment:
I think you might want to search for `r'^Intel RealSense ((\S+?)(\d+)(\S*))'` or something like that

Reply:
done